runner/plugins/odbc/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Change this to odbc.build and have it run through the build script like the other build files since not all Windows supports bash (my computer doesn't).

Reply:
hmm plugins don't work on windows though. So I guess we'll need to make a separate version for windows?

Reply:
maybe build constraints can help